config/webpack.prod.config.js

```diff
@@ -87,15 +86,19 @@ module.exports = Merge.smart(commonConfig, {
       },
     ],
   },
+  optimization: {
+    runtimeChunk: 'single',
+    splitChunks: {
+      cacheGroups: {
+        commons: {
+          name: 'common',
```

Comment:
Instead of putting all node_module packages into a `vendor.min.js`, this will put any chunk that is common in more than one entry into a `common.min.js`. This will require a change in the platform since we will need to load `common.min.js` instead of `vendor.min.js`.